pkg/auth/audit/audit.go

```diff
@@ -110,7 +111,7 @@ func newAuditLog(writer *LogWriter, req *http.Request) (*auditLog, error) {
 	contentType := req.Header.Get("Content-Type")
 	loginReq := isLoginRequest(req.RequestURI)
 	if writer.Level >= levelRequest || loginReq {
-		if bodyMethods[req.Method] && contentType == contentTypeJSON {
+		if bodyMethods[req.Method] && (contentType == contentTypeJSON || contentType == contentTypeJSONCharset) {
```

Comment:
Any reason why we can't just check the prefix?
```suggestion
		if bodyMethods[req.Method] && strings.HasPrefix(contentType, contentTypeJSON) {
```

Reply:
Can do that, dint want to change it since this code seems to be existing for quite a while. But makes sense to check prefix.

Reply:
I agree with @aiyengar2's suggestion.